### DISCUSSION TITLE
raw data の訳語

### DISCUSSION THREAD

**Discussion by tsubasakat** [stzn/the-swift-programming-language-jp#307] [General]
`raw data` を訳したいのですが、`生データ` と `原始データ` でどちらがいいでしょうか。
他にも何か意見をもらえたらありがたいです。

---
これは [Advanced Operators](https://github.com/stzn/the-swift-programming-language-jp/blob/5aeba4cd87290c4080f9410cc8023f4b42677fb0/language-guide/advanced-operators.md) のページにあった訳語についてです。
そこでは "[the individual raw data bits within a data structure](https://docs.swift.org/swift-book/LanguageGuide/AdvancedOperators.html#ID29)" を「[データ構造内の個々のデータのビット](https://github.com/stzn/the-swift-programming-language-jp/blob/5aeba4cd87290c4080f9410cc8023f4b42677fb0/language-guide/advanced-operators.md?plain=1#L16)」としています。

**Comment by stzn** [comment]
> raw data

難しいですね...個人的には「元データ」がしっくりきますねー。

> データ構造内の個々のデータのビット

この訳大変わかりやすいと思います。

こういった単語は状況に応じて訳がだいぶ変わるので、個別に対応するのが良さそうな気がしますねー。

**Comment by tsubasakat** [comment]
言われてみると、これは確かに文脈で変わりますね…。
おっしゃるとおりで、一語に縛るのはよくなさそうです。
臨機応変します。
